frontend/lib/ui/ProgressList/ProgressList.module.css

```diff
@@ -5,18 +5,20 @@
   width: 19.5rem;
 
   li {
-    padding: 0.625rem 0.75rem;
     height: 2.75rem;
     border-radius: 0.5rem;
     display: flex;
     align-items: center;
+    gap: 0.5rem;
+    &:global(:hover) {
+      background-color: var(--color-hover);
+    }
 
     aside {
       display: flex;
       flex: 0 0 32px;
-      svg {
-        width: 24px;
-      }
```

Comment:
This caused https://github.com/kiesraad/abacus/pull/252#pullrequestreview-2235045347